### PR TITLE
feat: Navbar revamp

### PR DESCRIPTION
**Pull request by ChefMomota**
---

## PR-Codex overview
This PR focuses on enhancing the menu functionality and UI components in the application. It introduces new sub-menu items, modifies existing components, and improves styling for better user experience.

### Detailed summary
- Added `activeSubItemChildItem` to multiple type definitions.
- Adjusted positioning in `InPageBanner.tsx`.
- Introduced `SubMenu` component in various views.
- Added new icons like `GameIcon`.
- Updated footer links for better navigation.
- Improved menu item rendering with child items support.

> The following files were skipped due to too many changes: `packages/uikit/src/__tests__/widgets/__snapshots__/menu.test.tsx.snap`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: packages/localization/src/config/translations.json [hunk 1]
```diff
@@ -3511,5 +3507,21 @@
   "PancakeSwap Quest-Beta Now Live": "PancakeSwap Quest-Beta Now Live",
   "Create Your Profile": "Create Your Profile",
   "Pancake4ever! Celebrate PancakeSwap's 4th Birthday with us!": "Pancake4ever! Celebrate PancakeSwap's 4th Birthday with us!",
-  "Join Now": "Join Now"
+  "Join Now": "Join Now",
+  "Perps": "Perps",
+  "Farm / Liquidity": "Farm / Liquidity",
+  "Stargate": "Stargate",
```
[the remark this answers is not in the document]
Comment (ChefMomota) [reply packages/localization/src/config/translations.json:3514]: We need add those if not the test will fail.

File: packages/uikit/src/components/DropdownMenu/DropdownMenu.tsx [hunk 2]
```diff
@@ -1,26 +1,148 @@
...
+  const [isSubMenuOpen, setIsSubMenuOpen] = useState(true);
+  const { isMobile, isMd } = useMatchBreakpoints();
+  const { type = DropdownMenuItemType.INTERNAL_LINK, label, href = "/", status, disabled, items, ...itemProps } = item;
+  const hasChildItems = useMemo(() => Boolean(items && items.length > 0), [items]);
+
+  const isActive = useMemo(() => {
+    return Boolean(isChildItems ? item.href === activeSubItemChildItem : item.href === activeItem);
+  }, [item, isChildItems, activeSubItemChildItem, activeItem]);
+
+  const MenuItemContent = (
```
[the remark this answers is not in the document]
Comment (ChefMomota) [reply packages/uikit/src/components/DropdownMenu/DropdownMenu.tsx]: done

File: apps/aptos/components/Menu/footerConfig.ts [hunk 3]
```diff
@@ -88,27 +88,39 @@ export const footerLinks: (t: ContextApi['t']) => FooterLinkType[] = (t) => [
         label: t('Documentation'),
         href: 'https://docs.pancakeswap.finance/',
       },
+      {
+        label: t('Audits'),
+        href: 'https://docs.pancakeswap.finance/readme/audits',
+      },
+      {
+        label: t('Legacy products'),
+        href: '/', // TODO
```
Comment (ChefMomota) [inline apps/aptos/components/Menu/footerConfig.ts:97]: Update it before merge

File: packages/uikit/src/widgets/Menu/components/footerConfig.ts [hunk 4]
```diff
@@ -88,27 +84,39 @@ export const footerLinks: (t: ContextApi["t"]) => FooterLinkType[] = (t) => [
         label: t("Documentation"),
         href: "https://docs.pancakeswap.finance/",
       },
+      {
+        label: t("Audits"),
+        href: "https://docs.pancakeswap.finance/readme/audits",
+      },
+      {
+        label: t("Legacy products"),
+        href: "/",
```
Comment (ChefMomota) [inline packages/uikit/src/widgets/Menu/components/footerConfig.ts:93]: Update it before merge

**Review by Chef-Yogi — APPROVED** [review]

**Pull request merged**